htlcswitch/switch.go

```diff
@@ -736,9 +736,8 @@ func (s *Switch) addLink(link ChannelLink) error {
 		return err
 	}
 
-	log.Infof("Added channel link with chan_id=%v, short_chan_id=(%v), "+
-		"bandwidth=%v", link.ChanID(), spew.Sdump(link.ShortChanID()),
-		link.Bandwidth())
+	log.Infof("Added channel link with chan_id=%v, short_chan_id=(%v)",
```

Comment:
What's the rationale behind this change? Simply removing a portion of the logging message isn't a fix. The internal goroutines of the link should be modified to ensure that the `Bandwidth` method can be queried even when the channel is waiting for re-establishment.